_includes/header.html

```diff
@@ -31,7 +31,7 @@
       <div class="container">
         <a class="bs-docs-booticon" href="/" aria-label="Bootstrap">
         {% include bootstrap-outline.svg %}</a>
-        <h1>The Bootstrap Blog</h1>
+        <h1><a href="/">The Bootstrap Blog</a></h1>
```

Comment:
Is this really needed? We wrap the whole thing already in an `a` tag.

Reply:
Only the `<svg>` is linked currently.

Reply:
Consolidated the anchors so it's one linke.

Reply:
Ah yeah you are right. Your call how you want the hover effect to be then, either way is fine.